### PR TITLE
Lvnod 27 create model user

### PR DESCRIPTION
**Pull request by yarvanya**
I have created model of user and profile with migration and seeders.

### CODE REVIEW

File: server/config/env.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  PORT: process.env.PORT || 8090,
+  DATABASE_URL: process.env.DATABASE_URL || 'jdbc:postgresql://localhost:5432/dami_database',
+  DATABASE_NAME: process.env.DATABASE_NAME || 'dami_database',
+  DATABASE_HOST: process.env.DATABASE_HOST || 'localhost',
+  DATABASE_USERNAME: process.env.DATABASE_USERNAME || 'postgres',
+  DATABASE_PASSWORD: process.env.DATABASE_PASSWORD || 'admin',
+  DATABASE_PORT: process.env.DATABASE_PORT || 5432,
+  DATABASE_DIALECT: process.env.DATABASE_DIALECT || 'postgres',
+
+  NODE_ENV: process.env.NODE_ENV || 'development',
```
Comment (denisbodnar) [inline server/config/env.js]: Remove comma after last property

**Review by denisbodnar — DISMISSED** [review]

File: server/config/env.js (hunk 1 above)
Comment (yarvanya) [reply server/config/env.js]: Done

File: server/router/routes/users.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  app.get('/users', (req, res) => {
+    db.users.findAll()
+      .then(users => {
+        res.json(users);
+      });
+  });
+
+  app.get('/users/:id', (req, res) => {
+    const id = req.params.id;
+    console.log(id);
```
Comment (lhalam) [inline server/router/routes/users.js]: remove `console.log()`

File: server/models/profile.model.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      allowNull: false
+    },
+    user_id: {
+      type: DataTypes.INTEGER
+    },
+  }, {
+    timestamps: false,
+    paranoid: true,
+    underscored: true
+
```
Comment (lhalam) [inline server/models/profile.model.js]: remove empty line

File: server/config/env.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
```
Comment (lhalam) [inline server/config/env.js]: remove the one empty line

File: server/config/env.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+const env = {
+  PORT: process.env.PORT || 8090,
+  DATABASE_URL: process.env.DATABASE_URL || 'jdbc:postgresql://localhost:5432/dami_database',
+  DATABASE_NAME: process.env.DATABASE_NAME || 'dami_database',
+  DATABASE_HOST: process.env.DATABASE_HOST || 'localhost',
+  DATABASE_USERNAME: process.env.DATABASE_USERNAME || 'postgres',
+  DATABASE_PASSWORD: process.env.DATABASE_PASSWORD || 'admin',
+  DATABASE_PORT: process.env.DATABASE_PORT || 5432,
+  DATABASE_DIALECT: process.env.DATABASE_DIALECT || 'postgres',
+
```
Comment (lhalam) [inline server/config/env.js]: remove the empty line

File: server/models/profile.model.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+   const Profile = sequelize.define('profile', {
+    ID: {
```
Comment (lhalam) [inline server/models/profile.model.js]: set the correct number of spaces, 
use 2 spaces

File: app.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+   next();
+  });
+
+router(app, db);
+
+db.sequelize.sync().then(() => {
+   app.listen(PORT, () => {
+     console.log('Express listening on port:', PORT);
+   });
+
```
Comment (lhalam) [inline app.js:42]: remove the empty line

File: server/models/user.model.js [hunk 8]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
```
Comment (lhalam) [inline server/models/user.model.js]: set the correct number of spaces, 
use 2 spaces

File: server/models/user.model.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+const bcrypt = require('bcrypt');
+
+module.exports = (sequelize,DataTypes) => {
+    const User = sequelize.define('user',{
```
Comment (AlbaMontis) [inline server/models/user.model.js]: Missed space after 'user'

File: server/models/user.model.js [hunk 10]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+
+module.exports = (sequelize,DataTypes) => {
```
Comment (AlbaMontis) [inline server/models/user.model.js]: Missed space between 'sequelize,DataTypes'

File: server/models/user.model.js [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+          type: DataTypes.INTEGER
+        }, 
+        is_activate:{ 
+          type: DataTypes.BOOLEAN, 
+          allowNull: false 
+        }, 
+        status_user_state: { 
+            type: DataTypes.STRING 
+        } 
+        },{
```
Comment (AlbaMontis) [inline server/models/user.model.js]: Missed space between },{

File: app.js [hunk 12]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const express = require('express'),
+  bodyParser = require('body-parser'),
```
Comment (AlbaMontis) [inline app.js]: Use one variable per declaration.

**Pull request closed without merging**